.github/workflows/continuous-integration.yml

```diff
@@ -183,6 +184,9 @@ jobs:
           - "11"
           - "12"
           - "13"
+        include:
+          - php-version: "8.0"
+            postgres-version: "12"
```

Comment:
Why not the latest version?

Reply:
My intention was to not slow the CI down by running tests for all Postgres-Versions with php 8 again. But I don't have strong opinions on which Postgres version to choose here. We can change this.